### PR TITLE
Missing runtimelib urijs

### PR DESCRIPTION
**Pull request by legander**
Remove urijs and form-urlencoded dependencies in favor of URL/URLSearchParams web api.

url.js is a bundle with https://github.com/zloirock/core-js/blob/master/packages/core-js/web/url.js + https://github.com/zloirock/core-js/blob/master/packages/core-js/web/url-search-params.js and is intended to be hosted on https://jslib.k6.io

Fixes issue https://github.com/loadimpact/har-to-k6/issues/59 the script tries to use a runtime dependency that is not available.
Resolves https://github.com/loadimpact/har-to-k6/issues/4

Also addresses https://github.com/loadimpact/har-to-k6/pull/10/files.
_Note that form-urlencoded serializes multi value query params differently than URLSearchParams.toString()_

### CODE REVIEW

File: src/aid.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ function extrinsic(enumeration) {
   return Math.max(...Object.values(enumeration)) + 1
 }
 
-function nought(value) {
+function isNil(value) {
```
Comment (allansson) [inline src/aid.js:33]: Nooooooo!

Comment (allansson) [reply src/aid.js:33]: What's next? Removing `const undef = void 0`? This is a downward spiral!

**Review by allansson — APPROVED** [review]
For to make great progress!

**Comment by allansson** [comment]
ESLint is puking a bunch of errors... better fix those before merging 😅

**Comment by legander** [comment]
Think it is because of the url.js bundle which is suppose to end up in jslib @allansson

**Pull request merged**